contracts/src/CrossChainApplications/NativeTokenBridge/README.md

```diff
@@ -0,0 +1,34 @@
+# Native-to-Native Token Bridge
+
+A pair of smart contracts built on top of Teleporter to support using an ERC20 token or the native token of any `subnet-evm` chain as the native token for a given subnet.
+
+## Design
+The native-to-native bridge is implemented using two primary contracts.
+### `NativeTokenSource`
+    - Lives on the Source chain. Pairs with exactly one `NativeTokenDestination` contract on a different chain.
```

Comment:
We should define what the "source chain" and "destination chain" are in this context above in this README